app/xslt/ead_remove_namespace.xsl

```diff
@@ -0,0 +1,41 @@
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
```

Comment:
If this is essentially borrowed from https://github.com/cdlib/dsc-oac-voro/blob/master/xslt/Remove-Namespaces.xsl, should we comment that?

Reply:
Good call. Added a comment.